components/national-team-profile.js

```diff
@@ -0,0 +1,343 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import dynamic from 'next/dynamic';
+
+import { object, string, number } from 'yup';
+import {
+  Grid,
+  Flex,
+  Select,
+  Heading,
+  Text,
+  Switch,
+  Link,
+} from '@chakra-ui/react';
+import { CheckIcon } from '@chakra-ui/icons';
+
+import { buttonVariants } from 'components/prismic-wrapper';
+import Input from 'components/input'; // DO NOT DYNAMIC IMPORT, BREAKS FORMS
+import Textarea from 'components/textarea'; // DO NOT DYNAMIC IMPORT, BREAKS FORMS
+
+const PrismicWrapper = dynamic(() => import('components/prismic-wrapper'));
+const Label = dynamic(() => import('components/label'));
+const Button = dynamic(() => import('components/button'));
+const Container = dynamic(() => import('components/container'));
+const Required = dynamic(() => import('components/required'));
+const InlineError = dynamic(() =>
+  import('components/errors').then(({ InlineError }) => InlineError)
+);
+
+import { api } from 'modules/api';
+import { rem } from 'styles/theme';
+
+const NATIONAL_TEAMS = ['England', 'Scotland', 'Wales'];
+const POSITIONS = ['Keeper/Chaser', 'Beater', 'Seeker'];
+
+const NationalTeamFormSchema = object().shape({
+  first_team: string()
+    .nullable()
+    .required('Please select your first choice National Team.'),
+  second_team: string().nullable(),
+  third_team: string().nullable(),
+  position: string().required('Please select your primary position.'),
+  playstyle: string().required(
+    'Please enter a short description of your playstyle.'
+  ),
+  years: number().required(
+    'Please estimate how many years of active quidditch experience you have.'
+  ),
+  experience: string().required('Please summarise your quidditch experience.'),
+});
+
+const handleFormSubmit = async (values, setServerError, setServerSuccess) => {
+  try {
+    setServerError(null);
+    setServerSuccess(null);
+
+    // TODO: Update and test api endpoint for this form.
+    await api.post('/contact/national', values);
+    setServerSuccess(true);
+  } catch (err) {
+    setServerError(err?.response?.data?.error?.message);
+  }
+};
+
+// TODO: Test this to recieving a data structure provided by the API.
+const NationalTeamProfileForm = ({ profile }) => {
```

Comment:
As profile is undefined (since we're not passing anything down to the component ATM) the defaultValues is causing builds to blowup (`can't read property national_team_interest of undefined`). For the moment let's add a fallback and that will at least keep things moving along:
```suggestion
const NationalTeamProfileForm = ({ profile = {} }) => {
```